README.md

```diff
@@ -64,13 +70,72 @@ You can split `ucm.lua` into multiple files and use Lua modules as usual. The on
 
 E.g. your scrpt is placed in `lua_dir`. So you can amalgamate your `ucm.lua` by the following command:
 ```
-docker run --rm -it -v $(pwd):/app enapter/rockamalg amalg -o ucm.lua -d deps lua_dir
+docker run --rm -it \
+-v $(pwd):/app \
+enapter/rockamalg \
+amalg -o ucm.lua -d deps lua_dir
 ```
-
 ## Server mode
 
 It's possible to run rockamalg in server mode. This mode is useful to integrate with another services, which works in separate Docker containers.
 
+## Dependency caching
+
+Downloading dependencies for each amalgamation could take a lot of time. By default rockamalg uses dependency cache to speedup the amalgamation.
+
+If you want to persist the cache between rockamalg runs, mount a volume into `/opt/rockamalg/.cache` in the container:
```

Comment:
Downloading dependencies for each amalgamation could take a lot of time. Rockamalg caches dependencies to speedup the amalgamation.

You need to mount a volume into `/opt/rockamalg/.cache` to reuse cache between runs.

Single run:
```
...
```

Server mode:
```
...
```